projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/breve/highlight/index.tsx

```diff
@@ -284,9 +285,11 @@ export function registerBreveHighlights() {
 				const { getIgnoredSuggestions, isFeatureEnabled, isProofreadEnabled } = globalSelect(
 					'jetpack/ai-breve'
 				) as BreveSelect;
+				const { getAiAssistantFeature } = globalSelect( 'wordpress-com/plans' );
+				const isFreePlan = getAiAssistantFeature().currentTier?.value === 0;
```

Comment:
couldn't we skip this `const` and simply call useBreveAvailability (as in line https://github.com/Automattic/jetpack/pull/38743/files#diff-7ca7a4761556dfae6cde5bb8df76daec871925d13a8bdbe76d94d503498705d2R61), which internally already checks `getAiAssistanteFeature` and uses `currentTier` value to pass as `isFreePlan` param to the underlying `getBreveAvailability` call?

Reply:
The problem here is that this is not a React component, so we can't use hooks. That is why `globalSelect` is used instead of `useSelect`, so I had to dance around that.